serverless/serverless-getting-started.adoc

```diff
@@ -13,23 +13,27 @@ include::modules/technology-preview.adoc[leveloffset=+1]
 
 [id="how-it-works_{context}"]
 == How {ServerlessProductName} works
-Developers on {ServerlessProductName} can use the provided Kubernetes-native APIs, as well as familiar languages and frameworks, to deploy applications and container workloads.
-For information about installing {ServerlessProductName}, see xref:../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[Installing OpenShift Serverless].
 
-{ServerlessProductName} on {product-title} enables stateful, stateless, and serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
+Developers on {ServerlessProductName} can use the provided Kubernetes native APIs, as well as familiar languages and frameworks, to deploy applications and container workloads.
+
+{ServerlessProductName} on {product-title} enables stateless, serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
 
 {ServerlessProductName} is based on the open source Knative project, which provides portability and consistency across hybrid and multi-cloud environments by enabling an enterprise-grade serverless platform.
 
-[id="knative-eventing_{context}"]
-== Knative Eventing
-A developer preview version of Knative Eventing is available for use with {ServerlessProductName}. However, this is not included in the {ServerlessOperatorName} and is not currently supported as part of this Technology Preview. For more information about Knative Eventing, including installation instructions and samples, see the link:https://openshift-knative.github.io/docs/docs/index.html[Knative Eventing on {product-title}] documentation.
+[id="serverless-ha_{context}"]
+== High availability in {ServerlessProductName}
+
+Active/passive high availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs.
+HA functionality is available in {ServerlessProductName} for the Knative Serving control plane.
+
+In a HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
+
+Active/passive HA in {ServerlessProductName} is available through leader election.
```

Comment:
I'd suggest 'acheived via' instead of 'available through', but you're the languager

Reply:
I wasn't sure about this one, so I checked the IBM style guide, which says:
"via prep
Use this term, which means “by way of,” to refer to routing data, such as in “Messages are routed via SMTP.” In other situations, try to replace “via” with one of the following synonyms: “across,” along,” “by,” “from,” “on,” or “through.”"

Let me check with OCP folks.

Reply:
+1 to "available through"

Reply:
+1 to "available through" (or "by way of")

Reply:
+1 to an option other than "via".